scripts/qa_crowbarsetup.sh

```diff
@@ -4646,7 +4646,7 @@ function onadmin_run_cct()
         ensure_packages_installed git-core gcc make ruby2.1-devel
 
         if [[ $cct_tests =~ ":ui:" ]]; then
-            ensure_packages_installed libqt4-devel libQtWebKit-devel xorg-x11-server xorg-x11-server-extra
+            ensure_packages_installed libqt4-devel libQtWebKit-devel xorg-x11-server xorg-x11-server-extra zlib-devel
```

Comment:
@aplanas @bmwiedemann Probably yes, or we could change the configuration in https://github.com/SUSE-Cloud/cct/blob/master/.bundle/config first to accept the gems installed via zypper .  I will look into packaging cct later.